Tasks/VsTestV2/distributedtest.ts

```diff
@@ -99,9 +99,14 @@ export class DistributedTest {
             if (this.inputDataContract.TestSelectionSettings.TestSelectionType.toLowerCase() !== 'testassemblies') {
                 sourceFilter = ['**\\*', '!**\\obj\\*'];
             }
-
+            ci.publishEvent({MinMatchLines: sourceFilter.length});
+            var start = new Date().getTime();
```

Comment:
use Date.now();

what does getTime return? Is it ticks since UTC start? If not this might have issues at day boundaries

Reply:
This returns the number of milliseconds since UTC. So it will not be a problem